src/guides/v2.4/graphql/queries/store-config.md

```diff
@@ -301,6 +321,16 @@ Attribute |  Data Type | Description | Example
 `root_category_id` | Int | The ID of the root category
 `title_separator` | String | Identifies the character that separates the category name and subcategory in the browser title bar | `-`
 
+### Supported Catalog attributes
```

Comment:
```suggestion
### Supported catalog attributes
```

Reply:
By the way, it's rather about the customer attributes than the catalog attributes

Reply:
**fixed**